src/main/java/de/tum/in/www1/artemis/service/AutomaticTextFeedbackService.java

```diff
@@ -60,14 +60,19 @@ public void suggestFeedback(@NotNull Result result) {
                             // Filter all other blocks in the cluster for those with Feedback
                             .filter(element -> feedbackForTextExerciseInCluster.containsKey(element.getId()))
 
+                            // sort, kill duplicates
                             // Find the closest block
```

Comment:
I think it would be better if this was in one line, it might just be me, but it seems like one of the comments was unintentional or outdated